modules/task/task.go

```diff
@@ -98,10 +87,3 @@ type Config struct {
 	broker  string `yaml:"broker"`
 	timeout string `yaml:"timeout"`
 }
-
-type backendRegisterFn func(service.Host, Config) (Backend, error)
-
-// RegisterAsyncBackend registers the backend for the async task module
-func RegisterAsyncBackend(backendRegisterFn backendRegisterFn) {
```

Comment:
Could you provide more context, why are you removing it? It seems pretty standard across modules to have a registration functions?